.github/workflows/ci-ultra-tests.yml

```diff
@@ -1,11 +1,10 @@
-name: ULTRA CI
+name: ULTRA CI Base Tests
```

Comment:
I think we should keep changes specific to ULTRA separate from those changing SMARTS as a general case.